package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "webpack-ts-starter",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/node": "^18.11.10",
+    "@types/webpack": "^5.28.0",
+    "@typescript-eslint/eslint-plugin": "^5.45.0",
+    "@typescript-eslint/parser": "^5.45.0",
+    "autoprefixer": "^10.4.13",
+    "caniuse-lite": "^1.0.30001436",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "css-loader": "^6.7.2",
+    "cssnano": "^5.1.14",
+    "eslint": "^8.29.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsdoc": "^39.6.4",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-simple-import-sort": "^8.0.0",
+    "file-loader": "^6.2.0",
+    "html-webpack-plugin": "^5.5.0",
+    "mini-css-extract-plugin": "^2.7.1",
+    "node-sass": "^8.0.0",
+    "postcss": "^8.4.19",
+    "postcss-cssnext": "^3.1.1",
+    "postcss-import": "^15.0.1",
+    "postcss-loader": "^7.0.2",
+    "prettier": "^2.8.0",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "sugarss": "^4.0.1",
+    "ts-loader": "^9.4.2",
+    "ts-node": "^10.9.1",
+    "tsconfig-paths-webpack-plugin": "^4.0.0",
+    "typescript": "^4.9.3",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.0",
+    "webpack-dev-server": "^4.11.1",
+    "webpack-merge": "^5.8.0"
+  },
+  "scripts": {
+    "serve": "webpack serve --mode=development --progress",
+    "build": "webpack",
+    "build:prod": "webpack --config webpack.config.prod.ts"
+  },
```

Comment:
Eslint was added to the project, but it seems you didn't use it. Adding `"lint": "eslint src"` will allow you to run a check, automatically fix some errors and see other errors